ql/src/java/org/apache/hadoop/hive/ql/io/orc/encoded/EncodedTreeReaderFactory.java

```diff
@@ -1078,6 +1078,7 @@ public void nextVector(ColumnVector previousVector, boolean[] isNull, final int
         FilterContext filterContext, ReadPhase readPhase) throws IOException {
       if (vectors == null) {
         super.nextVector(previousVector, isNull, batchSize, filterContext, readPhase);
+        previousVector.isRepeating = false;
```

Comment:
@dongjoon-hyun /  @cxzl25  can you help confirm, hardcoding `isRepeating` here would be correct workaround for [ORC-1741](https://issues.apache.org/jira/browse/ORC-1741) & it won't lead to any other side affects?